.circleci/config.yml

```diff
@@ -65,6 +65,7 @@ jobs:
           name: Update pyenv
           command: |
             cd /opt/circleci/.pyenv/plugins/python-build/../.. 
+            git checkout master
```

Comment:
Seems with the new image, pyenv comes with the HEAD at v2.2.4:
```
circleci@default-12f749eb-c35b-4e89-8e46-dff291662509:/opt/circleci/.pyenv$ git status
HEAD detached at v2.2.4
```

Added a checkout to remedy this.

Reply:
This update and install step adds 2-3 mins. The new image comes iwth 2.7.18. Wonder if we could just use that version and drop this install.

Reply:
the version of pyenv? looks like latest is 2.2.5.
```
jian@artsy:~/internetcode/pyenv$ git tag | grep v2.2 | tail -1
v2.2.5
```
so perhaps 2.2.4 is fine.